guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p><code>make_column_transformer</code> is for applying column-wise transforms steps. <s>using which</s> <u>With this</u> the user can select any scaler function from scikit-learn <s>for</s> <u>to </u>transform<s>ing</s> the fields using the selected scaler. Suppose we want to use scaler1 on column c1 &amp; c2, and scaler2 on c3 &amp; c4, then we can use it as follows: </p><p><code>preprocessors = make_column_transfomer((scaler1, [c1,c2]), (scaler2,[c3,c4]))</code> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>